layouts/tag.njk

```diff
@@ -0,0 +1,37 @@
+{% extends "layouts/base.njk" %}
+
+{# Create breadcrumbs manually as tag should be excluded from collections #}
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    classes: "govuk-!-display-none-print",
+    items: [{
+      href: options.parentSite.url,
+      text: options.parentSite.name
+    }, {
+      href: options.pathPrefix,
+      text: options.homeKey
+    }, {
+      href: options.pathPrefix + "tags",
+      text: "Tags"
+    }]
+  }) if options.parentSite }}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    {{ appDocumentHeader({
+      title: "Posts tagged ‘" + tag + "’" | smart,
+      description: description
+    }) }}
+
+    {{ appProseScope(content) if content }}
```

Comment:
I don't think the description or content would ever get populated, at least using the current code which builds the collection? Should we drop it?

```suggestion
    {{ appDocumentHeader({
      title: "Posts tagged ‘" + tag + "’" | smart
    }) }}
```

Reply:
Happy to.

Reply:
Done.